aim/web/ui/src/components/kit/Input/Input.tsx

```diff
@@ -121,6 +132,12 @@ function InputWrapper({
     messagesFormatter(errorsMessages);
   }, [errorsMessages]);
 
+  React.useEffect(() => {
+    if (isValid !== undefined) {
```

Comment:
let's use the` _.isUndefined` method of lodash as we are using it on the 201 line